src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Flow.java

```diff
@@ -3505,6 +3508,9 @@ record RecordPattern(Type recordType, int _hashCode, Type[] fullComponentTypes,
 
         public RecordPattern(Type recordType, Type[] fullComponentTypes, PatternDescription[] nested) {
             this(recordType, hashCode(-1, recordType, nested), fullComponentTypes, nested);
+            if ("test.Test.R(test.Test.C _, test.Test.R(test.Test.C _, test.Test.R(test.Test.C _, test.Test.C _, test.Test.C _), test.Test.I _), test.Test.I _)".equals(toString())) {
```

Comment:
sorry missed this, this seems like testing code that should be removed right?

Reply:
Oops, sorry, will fix.